packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -871,19 +871,27 @@ Content-Type: application/json
-You should send the device fingerprint only if the trusted app has a computed fingerprint for the end user's client.
+Specifying your own device fingerprint is a highly privileged operation that is limited to trusted web applications and requires making authentication requests with a valid API token. You should send the device fingerprint only if the trusted app has a computed fingerprint for the end user's client.
 
-**Notes:**
+> **Note:** Device fingerprint is different from the device token. The time and device based MFA in the Okta Sign-On policy rules depends on the device token only and not on the device fingerprint. To read more about the device token, see [Context Object](#context-object). The time and device based MFA would work only if you pass the device token in the [client request context](/docs/reference/api-overview/#client-request-context).
 
-* Device fingerprint is different from the device token. The time and device based MFA in Okta SignOn policy rules depends on the device token only and not on the device fingerprint. To read more about the device token, see [Context Object](#context-object). The time and device based MFA would work only if you send the device token passed in the [client request context](/docs/reference/api-overview/#client-request-context).
-* To use device fingerprinting for the new or unknown-device email notification feature, include the `User-Agent` header in the request. For more information, see the [General Security documentation](https://help.okta.com/en/prod/Content/Topics/Security/Security_General.htm?).
+##### Device Fingerprint Best Practices
 
-##### Request example for device fingerprinting
+Use the following recommendations as guidelines for generating and storing a device fingerprint for the `X-Device-Fingerprint` header for web apps and native apps.
+
+**Web Apps**<br>
+Okta recommends using a secure, HTTP-only cookie with a random/unique value on the customer's domain as the default implementation. See [Cookie flags that matter](https://odino.org/security-hardening-http-cookies/#cookie-flags-that-matter) for more best practices on hardening HTTP cookies.
 
+**Native Apps**<br>
+Ask the device operating system for a unique device ID. See [Apple's information on DeviceCheck](https://developer.apple.com/documentation/devicecheck) for an example.
+
+* Okta strongly encourages you to use the [AppAuth pattern](https://tools.ietf.org/html/rfc8252) (system browser) for Native App sign-in flows. When you use the AppAuth pattern, you can then leverage Okta's device token implementation without any custom code. This allows an untrusted application to still provide a proof-of-device attestation and leverage Okta's device behavior and per-device MFA policies. Also, it allows the App to support new factors and functionality without having to change code.
+* If you choose to build your own sign-in UX for a Native App, you must configure the App to use a trusted backend to pass a device token.
```

Comment:
better wording: 
If you choose to build your own sign-in UI for a native application, the native application must use a trusted backend to pass the browser fingerprint to Okta. (would be great to refer to how make set a trusted backend here)

Reply:
Inline with my comment above (https://github.com/okta/okta-developer-docs/pull/1049/files#r393387338), I think we should take this sentence out too.